README.md

```diff
@@ -140,22 +145,22 @@ Deploy Azure IoT Hub and configure service communication to Azure Storage for Fi
 
 To validate a client device on public Internet can leverage file upload with a custom domain mapping, you will use two Terminal windows to run a client and a server .NET sample app.
 
-Run the File Upload notifications server app
+Run the File Upload notifications server application:
 
-* In a new terminal `CD` into the directory `./src/ServerSideFileNotification/`.
-* Build the .NET app: `dotnet build`
-* Run the sample app and leave it running. You will need to pass in the Service connection string from the `./deploy/quickstart-add-iothub.sh` script.
+1. In a new terminal `CD` into the directory `./src/ServerSideFileNotification/`.
+2. Build the .NET app: `dotnet build`.
+3. Run the sample application. You will need to pass in the Service connection string from the `./deploy/quickstart-add-iothub.sh` script.
 
 ```bash
 dotnet run "<iot hub service connection string>"
 ```
 
-* Leave this running.
+4. Leave this application running.
 
-Run the IoT Client sample app
+Run the IoT Client sample application:
 
-* In a new bash terminal `CD` into the directory `./src/SampleIoTClientFileUpload/`.
-* Prepare a `.env` file for the required variables:
+1. In a new bash terminal `CD` into the directory `./src/SampleIoTClientFileUpload/`.
+2. Prepare a `.env` file for the required variables:
 
 ```bash
```

Comment:
Can you indent this over 3, I couldn't through suggestions because the area wasn't modified.

Reply:
Can you please confirm it's the right indent you meant? I think it was for the code block. I also found a second code block which I indented and a one ordered list item set to 1 as well.

Reply:
Sorry, I see I wasn't clear. I meant the code block needs to be shifted under the numbered item. GitHub wouldn't let me add the suggestion since you didn't modify this area

Reply:
Thanks, I have now reviewed any code blocks within ordered lists and they all have 3 spaces in normal list, and 6 spaces in sub-lists. There are two locations with no lists and those don't have an indent. Please let me know if that is what you were looking for.

Reply:
Had missed several of your comments, they should now all be addressed but with the many changes may I ask you to review the Readme again? Thanks.

Reply:
All good!